INSTALL.md

```diff
@@ -36,6 +36,55 @@ Run the `deployment/provision` script from the command line.
 
     curl -sS get.pimoroni.com/phatdac | bash
 
+## USB Microphone
+
+These steps may vary based on your audio set-up. If you use a USB microphone:
```

Comment:
I don't really know how we deal with audio setup. It varies depending on the audio devices you choose to you and so is difficult to automate.